### PR TITLE
[docs] Update README with expected Kernel Version

### PR DESCRIPTION
**Pull request by VaishnaviHire**
This commit ensures that we recommend in our docs the expected
kernel version to mitigate any issues while running Windows
workloads. The expected kernel version is 10.0.17763.1457
or earlier.

### CODE REVIEW

**Review by aravindhp — APPROVED** [review]
/approve

File: README.md [hunk 1]
```diff
@@ -54,6 +54,16 @@ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 
 `<zone>` should be replaced with a valid Azure availability zone like `us-east-1a`.
 
+`<image>` should be a WindowsServer image offering that defines the 2019-Datacenter-with-Containers SKU with version
+          17763.1457.2009030514 or later.Run the following command to list Azure image info
```
Comment (PratikMahajan) [inline README.md:58]: Not sure if it should be `later` or `earlier`? 
Should this be later ?

Comment (gmarkley-VI) [reply README.md:58]: Earlier

**Comment by gmarkley-VI** [comment]
Please change to an earlier kernel, not a later version. Nice catch @PratikMahajan

File: README.md (hunk 1 above)
Comment (PratikMahajan) [inline README.md:58]:
```suggestion
          17763.1457.2009030514 or later. Run the following command to list Azure image info:
```
to make it consistent with `machineset-aws.md`

**Review by PratikMahajan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (VaishnaviHire) [reply README.md:58]: Thanks for catching this @PratikMahajan . This needs to be changed in openshift [docs](https://docs.openshift.com/container-platform/4.6/windows_containers/windows-containers-release-notes.html) as well

File: docs/machineset-aws.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
 # Creating an AWS Windows MachineSet
 
 _\<windows_container_ami\>_ should be replaced with the AMI ID of a Windows image with a container run-time installed. 
+ You must use Windows Server 2019 with a version 10.0.17763.1457 or later. Run the following command to list AWS image info:
+  ```shell script
+  $ aws ec2 describe-images \
+    --filters Name=name,Values=Windows_Server-2019-English-Full-ContainersLatest-2020.09.09
```
Comment (mansikulkarni96) [inline docs/machineset-aws.md:7]: a '\' should be added at the end of this line too, to copy it as a whole command.

**Pull request merged**